app/gateway/2.6.x/install-and-run/docker.md

```diff
@@ -220,32 +279,41 @@ The steps involved in starting Kong in [DB-less mode](/gateway/{{page.kong_versi
     ]
     ```
 
-    Notice the `MountPoint` entry. We will need that path in the next step.
+    Notice the `MountPoint` entry. You will need that path in the next step.
+
+### Prepare your configuration file
 
-3. Prepare your declarative configuration file:
+1. Prepare your declarative configuration file in `.yml` or `.json` format.
 
-    The syntax and properties are described on the [Declarative Configuration Format] guide.
+    The syntax and properties are
+    described in the [Declarative Configuration format] guide. Add whatever core
+    entities (Services, Routes, Plugins, Consumers, etc) you need to this file.
 
-    Add whatever core entities (Services, Routes, Plugins, Consumers, etc) you need there.
+    This guide assumes you named the file `kong.yml`.
 
-    On this guide we'll assume you named it `kong.yml`.
+1.  Save your declarative configuration file inside the `MountPoint` path
+mentioned in the previous step. In the case of this
+guide, that would be `/var/lib/docker/volumes/kong-vol/_data/kong.yml`.
 
-    Save it inside the `MountPoint` path mentioned in the previous step. In the case of this
-    guide, that would be `/var/lib/docker/volumes/kong-vol/_data/kong.yml`
+### Start Kong Gateway in DB-less mode
 
-4. Run the following command to start a container with {{site.base_gateway}}.
+{% include_cached /md/admin-listen.md desc='long' %}
+
+1. Run the following command to start a container with {{site.base_gateway}}.
 
-   Although it's possible to start the Kong container with `KONG_DATABASE=off`, it is usually
-   desirable to also include the declarative configuration file as a parameter via the
-   `KONG_DECLARATIVE_CONFIG` variable name. To do this, we need to make the file
-   "visible" from within the container. Use the `-v` flag, which maps
-   the Docker volume to the `/usr/local/kong/declarative` folder in the container.
+   Although it's possible to start the Kong container with just
```

Comment:
Too much detail here imo. Don't explain `KONG_DECLARATIVE_CONFIG` is optional. They won't be successful without it, so we can gloss over the detail here and add more context if they're reading the db-less guide